tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderTests.cs

```diff
@@ -254,5 +254,58 @@ public void UpdateOrderItem_OrderItemNotFound_NoOrderChange()
 
             order.Should().BeEquivalentTo(expected);
         }
+
+        [Test]
+        public void CalculateCostPerYearRecurring_OrderItemCostTypeRecurring_ReturnsTotalOrderItemCost()
```

Comment:
These tests should really start with CalculateCostPerYear, not CalculateCostPerYearRecurring